### PR TITLE
feat(Farms): APR use TooltipText and show deatil of APR

### PR DESCRIPTION
**Pull request by Woody810**
image

### CODE REVIEW

File: src/views/Farms/components/FarmCard/ApyButton.tsx [hunk 1]
```diff
@@ -61,19 +62,45 @@ const ApyButton: React.FC<React.PropsWithChildren<ApyButtonProps>> = ({
     onPresentApyModal()
   }
 
+  const lpRewardsAPR = useMemo(() => Math.max(Number(displayApr) - apr, 0).toFixed(2), [displayApr, apr])
```
[the remark this answers is not in the document]
Comment (Woody810) [reply src/views/Farms/components/FarmCard/ApyButton.tsx]: Done.

**Comment by Woody810** [comment]
> In mobile tapping confuses between opening roi calculator and tooltip, maybe we should only show this tooltip on desktop wdyt?

I think it's a good idea, let me fix it.

**Comment by Chef-Yogi** [comment]
LGTM 👍
There's a special case for Boosted farm APR, I can discuss with PM based on this PR and take care of Boosted part

**Review by Chef-Yogi — APPROVED** [review]

File: src/views/Farms/components/FarmCard/ApyButton.tsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ export interface ApyButtonProps {
   lpRewardsApr?: number
   addLiquidityUrl?: string
   strikethrough?: boolean
+  useTooltipText?: boolean
```
Comment (Chef-Yogi) [inline src/views/Farms/components/FarmCard/ApyButton.tsx:28]: maybe you can use 
`const { isMobile } = useMatchBreakpointsContext()` 
to control should show we the tooltip details for mobile

Comment (Woody810) [reply src/views/Farms/components/FarmCard/ApyButton.tsx:28]: okay, wait a sec.

Comment (Woody810) [reply src/views/Farms/components/FarmCard/ApyButton.tsx:28]: IsMobile cannot achieve logic, so it is recommended to use this version.

**Pull request merged**